specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/rbac.json

```diff
@@ -68,47 +68,50 @@
             }
           }
         },
-        "x-ms-pageable": {
-          "nextLinkName": "nextLink"
-        },
         "x-ms-examples": {
-          "ListRoleDefinitions": {
-            "$ref": "./examples/ListRoleDefinitions-example.json"
+          "DeleteRoleDefinition": {
+            "$ref": "./examples/DeleteRoleDefinition-example.json"
           }
-        },
-        "x-ms-odata": "#/definitions/RoleDefinitionFilter"
-      }
-    },
-    "/{scope}/providers/Microsoft.Authorization/roleAssignments/{roleAssignmentName}": {
```

Comment:
It seems you deleted all the role assignment CRUD operations. Looking at the file sans diff, I don't see them anywhere. These still need to be supported. I.e., I expect this file to contain only additions for custom role definitions.

Reply:
unclear to me what happened but it should be fixed now